engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/BaseServerTest.scala

```diff
@@ -162,8 +162,7 @@ abstract class BaseServerTest
   val zioRuntime   = new ExecutionContextRuntime(testExecutor)
 
   val zioExec         = ZioExec(zioRuntime)
-  val sqlDatabase     = SqlDatabase(config.directories.suggestionsDatabaseFile)
-  val suggestionsRepo = new SqlSuggestionsRepo(sqlDatabase)(system.dispatcher)
```

Comment:
`SqlSuggestionsRepo` isn't removed by this PR. Is it intended to be used somehow?

Reply:
The PR is still in a draft state...